game/src/edit/lanes.rs

```diff
@@ -84,18 +84,7 @@ impl LaneEditor {
             Widget::custom_row(row).centered(),
             change_speed_limit(ctx, parent.speed_limit),
             Btn::text_fg("Change access restrictions").build_def(ctx, hotkey(Key::A)),
-            Widget::custom_row(vec![
```

Comment:
Giving up on the "revert" function for now. It was always kind of confusing how it would do lane type first, then direction, if possible. Especially since this dialog now conflates changes that happen to an individual lane and changes that happen to the whole road, I'd prefer having a fresh look at this dialog with Yuwen's help.